osgar/drivers/rosmsg.py

```diff
@@ -57,7 +57,8 @@ def parse_imu( data ):
 
     q0, q1, q2, q3 = orientation  # quaternion
```

Comment:
I would recommend naming the variables as qx, qy, qz and qw for clarity

Reply:
(outdated)